eds/plugin.py

```diff
@@ -70,3 +71,8 @@ def descendants(self) -> List[Plugin]:
             plugins += plugin.children
             plugins.append(plugin)
         return plugins
+
+
+PluginInterface.register(Plugin)
+assert issubclass(Plugin, PluginInterface)
+assert isinstance(Plugin({}), PluginInterface)
```

Comment:
I think the purpose of these assertions should be handled in a test, rather than living here. we want to confirm that we can create a plugin that inherits from the base plugin and not getting any exceptions about not implementing the abstract methods and attributes.

Reply:
yeah totally, I had them in there just for development. removing now